CHANGELOG.rst

```diff
@@ -2,10 +2,25 @@
 Changelog
 =========
 
+.. _changes_0.3.9:
+
+`Unreleased <https://github.com/hydrologie/xdatasets>`_ (latest)
+----------------------------------------------------------------
+
+Contributors:
+
+Changes
+^^^^^^^
+* No change.
+
+Fixes
+^^^^^
+* No change.
```

Comment:
I need to change the default behaviour here to comment this block out on release.
```suggestion
..
  .. _changes_0.3.9:

  `Unreleased <https://github.com/hydrologie/xdatasets>`_ (latest)
  ----------------------------------------------------------------

  Contributors:

  Changes
  ^^^^^^^
  * No change.

  Fixes
  ^^^^^
  * No change.
```